fluent/syntax/ast.py

```diff
@@ -246,10 +246,10 @@ def __init__(self, id, key, **kwargs):
         self.key = key
 
 class CallExpression(Expression):
-    def __init__(self, callee, args, **kwargs):
+    def __init__(self, callee, args=None, **kwargs):
         super(CallExpression, self).__init__(**kwargs)
         self.callee = callee
-        self.args = args
+        self.args = args if args is not None else []
```

Comment:
This can be written more succinctly as `args or []`. It is also the convention used by other classes in this file.